src/main/java/racingcar/Race.java

```diff
@@ -0,0 +1,49 @@
+package racingcar;
+
+public class Race {
+
+    public static void execute() {
+        Car[] cars = IOClass.getName();
+        if (!IOClass.checkCarName(cars)) {
+            System.out.println("자동차 이름은 5자를 초과할 수 없습니다.");
+            return;
+        }
+
+        Race race = new Race();
+        int times = IOClass.getTimes();
+        repeatGame(cars, times);
+        IOClass.printWinners(cars);
+        return;
+    }
+
+    public static int getMaxScore(Car[] cars) {
+        int maximum = 0;
+        for (int i = 0; i < cars.length; i++) {
+            maximum = Math.max(maximum, cars[i].getPosition());
+        }
+        return maximum;
+    }
+
+    public static void repeatGame(Car[] cars, int times) {
+        while ((times--) > 0) {
+            for (int i = 0; i < cars.length; i++) {
+                int randomNumber = Race.getRandomNumber();
```

Comment:
둘 다, Race 클래스 내부에 있기 때문에 바로 메서드를 호출해도 될 것 같아요! 👀
```suggestion
                int randomNumber = getRandomNumber();
```